vladiate/test/test_main.py

```diff
@@ -86,7 +86,7 @@ def test_run(monkeypatch):
     monkeypatch.setattr('vladiate.main.main', main)
     try:
         monkeypatch.setattr('__builtin__.exit', exit)
-    except:
```

Comment:
I think this is actually an `ImportError`, this is why the tests are failing I believe.

Reply:
That's interesting because [I found something else in the docs](https://docs.pytest.org/en/latest/monkeypatch.html#_pytest.monkeypatch.MonkeyPatch.setattr). But whatever...